src/client/entity-editor/series-section/series-editor.tsx

```diff
@@ -0,0 +1,193 @@
+ *        relationship types for any pair of entities.
+ * @param {string} props.seriesType - Enity type of the series.
+ * @param {Function} props.onRemove - A function to be called when the Remove
+ *        button is clicked.
+ * @param {Function} props.onEdit - A function to be called when attribute values is
+ *        modified.
+ * @param {Function} props.onAdd - A function to be called when the add button
+ *        is clicked.
+ * @param {Array} props.seriesItemsArray - The list of entities.
+ *
+ * @returns {ReactElement} React Component as a modular section within the entity editor.
+ *
+ */
+function SeriesEditor({baseEntity, relationshipTypes, seriesType, onRemove, onAdd, onEdit, seriesItemsArray}:SeriesItemsProps) {
+	const [seriesItem, setSeriesItem] = useState(null);
+	const [targetEntity, setTargetEntity] = useState(null);
+
+	const handleEntityChange = (value: EntitySearchResult) => {
+		setTargetEntity(value);
+		// Convert "value" of type EntitySearchResult to type Entity
+		const otherEntity = {
+			bbid: _.get(value, 'id'),
+			defaultAlias: {
+				name: _.get(value, 'text')
+			},
+			disambiguation: _.get(value, 'disambiguation'),
+			type: _.get(value, 'type')
+		};
+		const relationships = generateRelationshipSelection(
+			relationshipTypes, baseEntity, otherEntity
+		);
+		if (relationships.length) {
+			// Filter out relationship type 70 - 74
+			const filterRelationships = relationships.filter(relationship =>
+				relationship.relationshipType.id > 69 && relationship.relationshipType.id < 75);
+			setSeriesItem(filterRelationships[0]);
+		}
+	};
+
+	const handleAdd = () => {
+		if (onAdd && targetEntity) {
+			// Set attributes value before adding the entity
+			seriesItem.attributes = [{attributeType: 1, value: {textValue: null}}, {attributeType: 2, value: {textValue: null}}];
+			if (seriesItem) {
+				onAdd(seriesItem);
+				// After adding the entity, clear the Entity Search Field by setting targetEntity to null.
+				setTargetEntity(null);
+			}
+		}
+	};
+
+	const handleNumberAttributeChange = (rowID, {target}) => {
+		const value = target.value === '' ? null : target.value;
+		const attributeNumber = {
+			attributeType: 2,
+			value: {textValue: value}
+		};
+		const attributePosition = {
+			attributeType: 1,
+			value: {textValue: null}
+		};
+		onEdit([attributePosition, attributeNumber], rowID);
+	};
+	return (
+		<div>
+			<h2>What {seriesType}s are part of this Series?</h2>
+			{seriesItemsArray.length ?
+				<>
+					{seriesItemsArray.map((value) => (
+						<Row className="margin-top-d5" key={`${value.rowID}`}>
+							<Col className="text-right" md={1}/>
+							<Col md={2}>
+								<input
+									className="form-control"
+									type="text"
+									value={_.find(value.attributes, {attributeType: 2})?.value.textValue || ''}
+									onChange={handleNumberAttributeChange.bind(this, value.rowID)}
+								/>
+							</Col>
+							<Col md={7}>
```

Comment:
The addition of the `form-control-static` CSS class to this Col element or its child will align it vertically to match the `form-control` class of the input in the previous column.

To compare the margins with and without the class (with added red border to see it better):
![image](https://user-images.githubusercontent.com/6179856/129576112-a85f7a9f-9bfc-4fbc-81e7-b49a7d58ce79.png)